packages/angular_devkit/architect/bin/architect.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Architect, Workspace } from '@angular-devkit/build-facade';
+import { JsonObject, dirname, normalize, tags } from '@angular-devkit/core';
+import { createConsoleLogger } from '@angular-devkit/core/node';
+import { existsSync, readFileSync } from 'fs';
+import * as minimist from 'minimist';
+import * as path from 'path';
+import { _throw } from 'rxjs/observable/throw';
+import { concatMap } from 'rxjs/operators';
+
+
+function findUp(names: string | string[], from: string) {
+  if (!Array.isArray(names)) {
+    names = [names];
+  }
+  const root = path.parse(from).root;
+
+  let currentDir = from;
+  while (currentDir && currentDir !== root) {
+    for (const name of names) {
+      const p = path.join(currentDir, name);
+      if (existsSync(p)) {
+        return p;
+      }
+    }
+
+    currentDir = path.dirname(currentDir);
+  }
+
+  return null;
+}
+
+/**
+ * Show usage of the CLI tool, and exit the process.
+ */
+function usage(exitCode = 0): never {
+  logger.info(tags.stripIndent`
+    architect build|run [project][:target][:configuration] [options, ...]
```

Comment:
We don't support `build` anymore, do we?

Reply:
Correct, removing.